Microsoft.Toolkit.Uwp.UI.Animations/ConnectedAnimations/ConnectedAnimationHelper.cs

```diff
@@ -114,22 +121,28 @@ void loadedHandler(object s, RoutedEventArgs args)
                     {
                         if (props.IsListAnimation && parameter != null && ApiInformationHelper.IsCreatorsUpdateOrAbove)
                         {
-                            props.ListViewBase.ScrollIntoView(parameter);
-
-                            // give time to the UI thread to scroll the list
-                            var t = props.ListViewBase.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, async () =>
+                            foreach (var listAnimProperty in props.ListAnimProperties)
                             {
-                                try
+                                if (listAnimProperty.ListViewBase.ItemsSource is IEnumerable<object> items && items.Contains(parameter))
                                 {
-                                    var success = await props.ListViewBase.TryStartConnectedAnimationAsync(connectedAnimation, parameter, props.ElementName);
+                                    listAnimProperty.ListViewBase.ScrollIntoView(parameter);
+
+                                    // give time to the UI thread to scroll the list
+                                    var t = listAnimProperty.ListViewBase.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, async () =>
```

Comment:
just out of curiosity what if props.ListAnimProperties has 10 items ? we will create 10 tasks to do the job ? isn't overkilling ?